recipes/pylint-flask/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pylint-flask" %}
+{% set version = "0.3" %}
+{% set sha256 = "7b0db2e98d34c3760664ab26464cd293d3736ee50999ad5f8313883c80c14d2f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - pylint-plugin-utils >=0.2.1
+    - pylint >=1.0
+    - astroid >=1.0
+    - logilab-common >=0.60.0
+
+test:
+  imports:
+    - pylint_flask
+
+about:
+  home: https://github.com/jschaf/pylint-flask
+  license: GPL 2
```

Comment:
License file not yet in an `sdist`. Proposing it be added in PR ( https://github.com/jschaf/pylint-flask/pull/5 ).